src/genie/libs/parser/iosxe/show_dmvpn.py

```diff
@@ -0,0 +1,124 @@
+# Metaparser
+import re
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Or, Optional
+
+
+# ==============================
+# Schema for 'show dmvpn'
+# ==============================
+class ShowDmvpnSchema(MetaParser):
+    """
+    Schema for 'show dmvpn'
+    Schema for 'show dmvpn interface <WORD>'
```

Comment:
instead of <WORD> please do use, <interface> again for sake of integrity of this code with our other codes